### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by the-meedu-app** [tancredi/fantasticon#16]
I used the next command `fantasticon my-icons/*.svg -o icon-dist` and I had the next error

```
Invalid option inputDir: my-icons/accessibility-outline.svg is not a directory
```

**Comment by tancredi** [comment]
Hi @the-meedu-app - differently from the previous version of the tool, you don't want to specify a glob for the input, just the directory. Try with `fantasticon my-icons -o icon-dist`

Let me know if that solves your problem.

**Comment by benbro** [comment]
The [README](https://github.com/tancredi/fantasticon#quick-usage) says:
```
fantasticon my-icons/*.svg -o icon-dist
```

**Comment by tancredi** [comment]
Oh, sorry about that @benbro - t'was a left over I've overseen.

Fixed in version `1.0.14`

**Comment by jonscottclark** [comment]
@tancredi Would it be possible as a user to provide a glob for `inputDir`, plus maybe a 2nd boolean option (`inputGlob: true` or something), to prevent `**/*.svg` appended to the end of the value provided for `inputDir`?
